docs/development/release.md

```diff
@@ -77,24 +77,24 @@ You will be prompted for a commit reference and an image tag. The commit referen
 * The [image release CI pipeline](https://github.com/ray-project/kuberay/blob/master/.github/workflows/image-release.yaml) also publishes the `github.com/ray-project/kuberay/ray-operator@vX.Y.Z-rc.0` Go module. KubeRay has supported Go modules since v0.6.0. Follow these instructions to verify the Go module installation.
     ```sh
     # Install the module. This step is highly possible to fail because the module is not available in the proxy server.
-    go install github.com/ray-project/kuberay/ray-operator@v0.5.0-rc.0
+    go install github.com/ray-project/kuberay/ray-operator@v1.0.0-rc.0
```

Comment:
It is weird that we say 'KubeRay has supported Go modules since v0.6.0.', but use v0.5.0-rc.0 as an example.